kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/DBExternalResource.java

```diff
@@ -27,7 +27,11 @@
     protected void after() {
 
         if (pds != null) {
-            pds.close();
+           try {
+               pds.close();
+           } catch (Exception e) {
+               
```

Comment:
Maybe knitpicking, but shouldn't we print at least a stack trace here? :)

Reply:
fixed